### ISSUE TITLE
Is the project still being worked on?

### ISSUE THREAD

**Issue by Diridibindy** [fork-maintainers/iceraven-browser#305]
As a user, I want this project to live so I have a question: is this still being worked on? Seeing how there have been no commits for a while and issues are left unanswered leaves me with little hope.

**Comment by CharmCityCrab** [comment]
The lead developer, interfect, tends to get things done in a flurry of activity about once a month.  A ton of issues get responses, a bunch of commits go through, and then there is a release, sometimes all in what looks like the span of a couple of hours if you're following along at home (Which isn't to say he isn't working on it privately in between, it's just that it isn't user-visible if he is).

It's a bit different from the cadence of a lot of projects, but the browser does get updated.  It's been about 20 days since the last update, but that isn't unusual for him.

He's indicated that he has a full-time job that has to do with coding (and it isn't this), and some other real life stuff he has to deal with regularly, and so the time he has to devote to this is the time he has to devote to this.

The good news for folks who would like more frequent updates and a wider array of features is that he is very open to, and in fact has asked for, other people to volunteer to help him code as project maintainers or just to submit the occasional patch or pull request.  When he briefly had a couple of other people helping him, we got much more frequent updates and a bunch of new features added, as well as more frequent merges from upstream.

So, your best bet if you want to see the project updated more frequently or to include new options that aren't just ported in from upstream, would be to find more developers to contribute to the project, or, if you know how to code, to contribute to the project yourself.  Even just someone to merge in minor releases with security patches and bug fixes from upstream, make sure everything works, and submit them as a pull request on a regular basis would be a big help.

I've always thought this would be the perfect thing for maybe a college student who is learning and trying to get some experience with Android app development and coding to hack on.  A browser is an interesting bit of software, there are plenty of user suggestions about options that could be added and the browser is in a sense here to add options (It's basically the options browser), so there is almost endless room to add features if one wants to, plus it would offer some experience in maintaining something that a lot of people use, in merging things from upstream with the code that's unique to Iceraven and making sure it all works together, troubleshooting, how to work in a group, etc.- even a bit of public relations!

The problem is finding more developers.  Even though I'm just (an admittedly very interested) user, I've tried to find developers and get them involved and mostly failed.  I'm not even sure where else I would look at this point.  However, that doesn't mean someone else couldn't find people- you guys may know places or people I don't, or be able to approach people in a way more likely to get them interested than the way I approach them.

It's also possible that a developer or two might just pop up- perhaps someone who discovers the browser as a user, sees a need, and wants to help out.

For now, though, don't panic. :)  Updates may not be as frequent as would be ideal, but the project hasn't been abandoned.  This is just the way things have worked in recent months.  Interfect will I would imagine find some time some day soon, address some issues, and put out an update when he has time.
